src/main/kotlin/org/neo4j/graphql/Translator.kt

```diff
@@ -66,9 +66,15 @@ class Translator(val schema: GraphQLSchema) {
                 // todo extract method or better object
                 val properties = properties(variable, fieldDefinition, propertyArguments(field))
```

Comment:
we can separate out all-properties and properties without-id which we can use later on in SET, UPDATE etc.

Reply:
Is it good for you if we manage that in the `setProperties` method by providing a list of properties to exclude?

Reply:
that's fine to, but then we should rename the method :)